pytest_terraform/tf.py

```diff
@@ -294,6 +313,7 @@ def create(self, request, module_dir):
             request.addfinalizer(self.tear_down)
         try:
             test_api = self.runner.apply()
+            self.config.hook.pytest_terraform_modify_state(tfstate=test_api)
```

Comment:
i think we still want to have this recieve a serialized state that we then reload. its probably worth writing a modifier hook that does something useful as example to compare the utility. also want to add this to the docs/readme.